src/APL/upstash-apl.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable class-methods-use-this */
+// eslint-disable-next-line max-classes-per-file
+import fetch, { Response } from "node-fetch";
+
+import { APL, AplReadyResult, AuthData } from "./apl";
+import { createAPLDebug } from "./apl-debug";
+
+const debug = createAPLDebug("UpstashAPL");
```

Comment:
It will resolve to `APL:UpstashAPL`, `APL` is already prefixed, so suffix is not necessary

Reply:
Wanted to maintain consistency with other APLs. Also wanted it to be the same as class name